composer.json

```diff
@@ -101,7 +101,7 @@
         "php": ">=7.4",
         "ext-json": "*",
         "dragonmantank/cron-expression": "^3.1",
-        "psr/log": "^1.0",
+        "psr/log": "^1.0 || ^3.0",
```

Comment:
Why not ^1|^2|^3 like Symfony Messenger: https://github.com/symfony/messenger/blob/1cc72a00521c69219ba79b7b04757d5e7fface66/composer.json#L20

Reply:
Messenger use the old syntax but yes, we can probably use 3 versions 🙂